qiskit/scheduler/methods/lowering.py

```diff
@@ -0,0 +1,115 @@
+# -*- coding: utf-8 -*-
```

Comment:
this file should probably be one directory up since it is not a scheduling "method"